helm/defectdojo/templates/celery-beat-deployment.yaml

```diff
@@ -11,6 +11,9 @@ metadata:
     helm.sh/chart: {{ include "defectdojo.chart" . }}
 spec:
   replicas: {{ .Values.celery.beat.replicas }}
+  {{- if .Values.revisionHistoryLimit }}
```

Comment:
Just final polishing, so  values by default parameters is disabled so then value is 10.  What is the reason with this approach you are not using `{{ .Values.revisionHistoryLimit  | default 10 }}`  to save 6 LoC  of flow control. It is not a big deal but code would be more readable